Documentation/dev/libvirt-howto.md

```diff
@@ -85,6 +85,35 @@ iptables -I INPUT -p tcp -s 192.168.124.0/24 -d 192.168.124.1 --dport 16509 \
   -j ACCEPT -m comment --comment "Allow insecure libvirt clients"
 ```
 
+If using `firewalld`, simply optain the name of the existing active zone which
+can be used to integrate the appropriate source and ports to allow connections from
+the IP range used by your cluster nodes. An example is shown below.
+
+```console
+sudo firewall-cmd --get-active-zones
```

Comment:
With `console` highlighting (which I think we want because you're showing output), you need [a `$` prompt here][1].

[1]: https://github.com/openshift/installer/pull/284#discussion_r218938492

Reply:
do you not want prompts on the others as well?

Reply:
> do you not want prompts on the others as well?

Prompts belong in `console` blocks (to differentiate between commands and output), but not in `sh` blocks (because they're not part of the shell input).  In this project, `sh` was more popular where we don't need to show output, so that's the pattern I'm trying to stick to (more on this in #27).